src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/LicenceController.kt

```diff
@@ -444,6 +445,46 @@ class LicenceController(
     )
   }
 
+  @Tag(name = Tags.LICENCES)
+  @PostMapping(value = ["/recently-approved"])
+  @PreAuthorize("hasAnyRole('CVL_ADMIN')")
+  @Operation(
+    summary = "Get a list of recently approved licence summaries matching the supplied list of prisons.",
+    description = "Get the recently approved licences matching the supplied list of prisons. Requires ROLE_CVL_ADMIN.",
+    security = [SecurityRequirement(name = "ROLE_CVL_ADMIN")],
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Returned matching licence summary details - empty if no matches.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            array = ArraySchema(schema = Schema(implementation = LicenceSummary::class)),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorised, requires a valid Oauth2 token",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Forbidden, requires an appropriate role",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+    ],
+  )
+  fun getRecentlyApprovedLicences(
+    @RequestBody body: RecentlyApprovedLicencesRequest,
```

Comment:
this will need an Valid annotation or the test will fail.. 

I could have made the test more clever and tried to look for validation annotations on the payload itself but its quite a bit more complex so just enforced an RequestBody to also have Valid

Reply:
fixed